### DISCUSSION TITLE
Feature request: cross-community publications

### DISCUSSION THREAD

**Discussion by allaway** [knowledgefutures/pubpub#2931] [Ideas]
It would be really awesome to be able to connect pubs to multiple communities. For example, my org and another org both have a pubpub community, it would be great to be able to list a pub we have co-authored with them on both pubpub wikis. 

A couple of thoughts: 

- we don't want to simply duplicate the pub, b/c we don't want to pollute crossref with multiple dois for the same thing
- a perhaps lighter-weight solution would be to allow pubs from any community to be listed in a collection. Then we could have a section on our pubpub for "collaborative pubs" that is a collection of our co-authored pubs that are hosted in different communities. 

Some add'l context on twitter: https://twitter.com/allawayr/status/1729929272796606693

Thanks for the great platform and for considering this!

**Comment by joedeville** [comment]
This would be really good and needed. Currently, we use PubPub for an overall site representing the work of a large and diverse community ([Copim](https://copim.pubpub.org/)), but some of this community have their own PubPub communities set up, and we would like to feature their pubs on our site, without republishing them (which would indeed cause a mess/confusion).

**Comment by tefkah** [comment]
Interesting idea! I think the lighter weight solution should be possible, here is a proof of concept of that

<img width="1203" alt="image" src="https://github.com/pubpub/pubpub/assets/21983833/673f9ab6-750a-4d38-a0a2-07270200efc2">

On the left a pub from demo.pubpub.org, on the right a pub from jtrialerror.pubpub.org, listed together in the same collection on demo.pubpub.org.

Can't promise anything regarding timelines, whether we will implement this, or whether this will scale at all (this makes the process of listing the pubs slightly less efficient, which may cause issues for larger layouts (such as copims front page))

**Reply by tefkah** [reply]
this will only have an effect on this collection, and won't (and shouldn't) show up in the collections on the other pub in e.g. the header 
<img width="400" alt="image" src="https://github.com/pubpub/pubpub/assets/21983833/4b2ca76b-5665-4b70-85ad-7b809a942c26">

**Reply by joedeville** [reply]
This would be perfect! Indeed, in our case, I don't think we'd need this showing up in the collection being pulled from. 

But yes, noted re no promises, timescale etc 🙂
